docs/changelog.txt

```diff
@@ -80,6 +80,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - `embark-assistant`: improved performance of surveying
 - `quickfort`: fixed eventual crashes when creating zones
 - `quickfort`: fixed library aliases for tallow and iron, copper, and steel weapons
+- `quickfort blueprint library <quickfort-library-guide>`: fix refuse stockpile config and prevent stockpiles from covering stairways in the quickfortress.csv library blueprint
```

Comment:
Links like this still don't work in the changelog, unfortunately (#1793). I'll probably just change this to "`quickfortress.csv` library blueprint" or something like that.